components/automate-ui/src/app/pages/+compliance/+node-credentials/node-credentials.module.ts

```diff
@@ -0,0 +1,40 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { RouterModule } from '@angular/router';
+import {
+  FormsModule,
+  ReactiveFormsModule
+} from '@angular/forms';
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+
+import { ChefComponentsModule } from '../../../components/chef-components.module';
```

Comment:
Should not use parent (..) notation in imports:
```suggestion
import { ChefComponentsModule } from 'app/components/chef-components.module';
```
Similar changes for other imports.